tools/serverpod_cli/lib/src/create/create.dart

```diff
@@ -94,363 +121,426 @@ Future<void> performCreate(
   var awsName = name.replaceAll('_', '-');
   var randomAwsId = math.Random.secure().nextInt(10000000).toString();
 
-  var projectDir = Directory(p.join(Directory.current.path, name));
-  if (projectDir.existsSync()) {
+  var serverpodDirs = ServerpodDirectories(
+    projectDir: Directory(p.join(Directory.current.path, name)),
+    name: name,
+  );
+  if (serverpodDirs.projectDir.existsSync()) {
     log.error('Project $name already exists.');
-    return;
+    return false;
   }
 
-  log.info(
-    'Creating project $name.',
-    style: const TextLogStyle(newParagraph: true),
-  );
+  if (template == ServerpodTemplateType.server) {
+    log.info(
+      '🚀 Creating Serverpod project "$name".',
+      type: TextLogType.init,
+    );
+  } else if (template == ServerpodTemplateType.module) {
+    log.info(
+      '📦 Creating Serverpod module "$name".',
+      type: TextLogType.init,
+    );
+  }
 
-  log.debug(
-    'Creating directory: ${projectDir.path}',
-    style: const TextLogStyle(
-      type: AbstractStyleType.bullet,
-    ),
-  );
-  projectDir.createSync();
+  bool success = await log.progress(
+      'Creating project directories.',
+      () => Future(() {
```

Comment:
Avoid using `=>` syntax when there is more than a single line (or very close to).

Reply:
The `Future` should be typed e.g. `Future<void>` (or if it's implicit, leave the `Future` out all togehter).